src/functionsDelete.ts

```diff
@@ -0,0 +1,41 @@
+import * as helper from "./functionsDeployHelper";
+import { Queue } from "./throttler/queue";
+import * as tasks from "./deploy/functions/tasks";
+import { DeploymentTimer } from "./deploy/functions/deploymentTimer";
+import { ErrorHandler } from "./deploy/functions/errorHandler";
+
+export async function deleteFunctions(
```

Comment:
In the future, I'd like to reuse this code during the normal functions deployment process too - I'm keeping it separate for now to avoid an ugly merge conflict with #3107